woocommerce/woocommerce
# Remove JSHint since we're using ESLint in commit hooks

### All Submissions:






### Changes proposed in this Pull Request:

Since we're using ESLint in our commit hooks, there's no reason to keep the JSHint configuration and tasks around. My IDE picked it up automatically and the two don't seem to agree on style. This PR removes the `.jshintrc` file and replaces the Grunt task for `jshint` with an `eslint` task. I've also removed the linting from the build process for now, as it'll cause a failure due to errors. This should be fine because our pre-commit hooks run eslint, so any changes in the file should result in that file becoming compliant (assuming the developer fixes the errors 😁)

### How to test the changes in this Pull Request:

1. Make sure you update your `node_modules` after checking out the PR!
2. Run `npm run build` and make sure there's no failure caused by ESLint
3. Run `grunt eslint` and see the output, should run ESLint

### Other information:
